packages/compass-settings/src/components/settings/privacy.tsx

```diff
@@ -34,86 +44,43 @@ const checkboxStyles = css({
   marginBottom: spacing[3],
 });
 
-const checkboxItems: CheckboxItem[] = [
-  {
-    name: 'autoUpdates',
-    label: (
-      <>
-        <Label htmlFor="autoUpdates">Enable Automatic Updates</Label>
-        <Description>
-          Allow Compass to periodically check for new updates.
-        </Description>
-      </>
-    ),
-  },
-  {
-    name: 'enableMaps',
-    label: (
-      <>
-        <Label htmlFor="enableMaps">Enable Geographic Visualizations</Label>
-        <Description>
-          Allow Compass to make requests to a 3rd party mapping service.
-        </Description>
-      </>
-    ),
-  },
-  {
-    name: 'trackErrors',
-    label: (
-      <>
-        <Label htmlFor="trackErrors">Enable Crash Reports</Label>
-        <Description>
-          Allow Compass to send crash reports containing stack traces and
-          unhandled exceptions.
-        </Description>
-      </>
-    ),
-  },
-  {
-    name: 'trackUsageStatistics',
-    label: (
-      <>
-        <Label htmlFor="trackUsageStatistics">Enable Usage Statistics</Label>
-        <Description>
-          Allow Compass to send anonymous usage statistics.
-        </Description>
-      </>
-    ),
-  },
-  {
-    name: 'enableFeedbackPanel',
+const checkboxItems: CheckboxItem[] = privacyFields.map((name) => {
+  const { short, long } = getSettingDescription(name);
+  return {
+    name,
     label: (
       <>
-        <Label htmlFor="enableFeedbackPanel">Give Product Feedback</Label>
-        <Description>
-          Enables a tool that our Product team can use to occasionally reach out
-          for feedback about Compass.
-        </Description>
+        <Label htmlFor={name}>{short}</Label>
+        {long && <Description>{long}</Description>}
       </>
     ),
-  },
-];
+  };
+});
+
+const settingStateLabels = {
+  'set-cli': (
+    <Banner variant={BannerVariant.Info} data-testid="set-cli-banner">
+      This setting cannot be modified as it has been set at Compass startup.
+    </Banner>
+  ),
+  'set-global': (
+    <Banner variant={BannerVariant.Info} data-testid="set-global-banner">
+      This setting cannot be modified as it has been set in the global Compass
+      configuration file.
+    </Banner>
+  ),
+  '': <></>,
```

Comment:
Just in case rendering fragments feels weird, null is a valid React child type

```suggestion
  '': null,
```

Reply:
Thanks, done in 57e7ff75dfaf2ba2268fd651f41eaef16d9ba858!